.azure-pipelines/pipelines.yml

```diff
@@ -415,9 +415,44 @@ stages:
               artifactName: windows.release
             condition: always()
 
-      - job: docker
+      - job: clang_cl
         dependsOn: ["release"]
```

Comment:
Question to @lizan and @envoyproxy/windows-dev maintainers; do we want to wait for msvc-cl "release" before beginning the clang-cl build? Or are there typically few enough modified components that we would have enough workers for both compilations at once, with minimal delay in seeing a release and then a docker result?

Reply:
I prefer having them in parallel.

Reply:
Done